ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx

```diff
@@ -11,18 +11,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import * as DOMPurify from 'dompurify';
+import { marked } from 'marked';
 import { Box } from '@mui/material';
 import { PanelProps } from '@perses-dev/plugin-system';
 import { MarkdownPanelOptions } from './markdown-panel-model';
 
 export type MarkdownPanelProps = PanelProps<MarkdownPanelOptions>;
 
+// Overall, this panel does:
+// 1. text --> html (with support for original markdown and GFM)
+// 2. sanitize the html to prevent XSS attacks
+
 export function MarkdownPanel(props: MarkdownPanelProps) {
   const {
     definition: {
       options: { text },
     },
   } = props;
 
-  return <Box sx={{ height: '100%', overflowY: 'auto' }}>{text}</Box>;
+  const html = marked.parse(text, { gfm: true });
+  const sanitizedHTML = DOMPurify.sanitize(html);
+
+  return <Box sx={{ height: '100%', overflowY: 'auto' }} dangerouslySetInnerHTML={{ __html: sanitizedHTML }} />;
```

Comment:
cool to see how `dangerouslySetInnerHTML` is used safely 😄

Reply:
I was kinda jealous she got to use it before I did. 😂

Reply:
living on the edge :)